GitUI/UserControls/BlameControl.cs

```diff
@@ -112,23 +112,22 @@ public async Task LoadBlameAsync(GitRevision revision, IReadOnlyList<ObjectId>?
                 return;
             }
 
+            int line = _clickedBlameLine is not null && _clickedBlameLine.Commit.ObjectId == objectId
+                ? _clickedBlameLine.OriginLineNumber
+                : initialLine ?? (fileName == _fileName ? BlameFile.CurrentFileLine : 1);
```

Comment:
A change in behavior:
Before the scroll position was kept when opening a new file or revision (unless the caller specified a line, for instance when going from file to file in FileTree). 
When switching files this was confusing, then it is clearer if the new file starts from line 1.
For the same file the scrollposition is the same as the line number. This is a simplification of the code, will probably require some hundreds of micro seconds extra.